pyccel/ast/literals.py

```diff
@@ -223,6 +223,8 @@ class Nil(Basic):
     class for None object in the code.
     """
 
-    def _sympystr(self, printer):
-        sstr = printer.doprint
-        return sstr('None')
\ No newline at end of file
+    def __str__(self):
+        """
+        return str ob and 'None' as value
+        """
```

Comment:
I don't think magic methods need docstrings. The `a.__str__()` method is just the method that is called when we do `str(a)` so the only docstring needed is the docstring of `str()`